omod/src/main/webapp/resources/scripts/services/roleService.js

```diff
@@ -0,0 +1,48 @@
+angular.module('roleService', ['ngResource'])
+	.config(function ($httpProvider) {
+	    // to prevent the browser from displaying a password pop-up in case of an authentication error
+	    $httpProvider.defaults.headers.common['Disable-WWW-Authenticate'] = 'true';
+
+	    // IE GET cache problem
+        if (!$httpProvider.defaults.headers.get) {
+            $httpProvider.defaults.headers.common = {};
+        }
+        $httpProvider.defaults.headers.common["Cache-Control"] = "no-cache";
+        $httpProvider.defaults.headers.common.Pragma = "no-cache";
+        $httpProvider.defaults.headers.common["If-Modified-Since"] = "0";
```

Comment:
If you put it in angular-common.js file in the uicommons module then you won't need to repeat